x-pack/test/ui_capabilities/security_and_spaces/tests/nav_links.ts

```diff
@@ -57,7 +57,8 @@ export default function navLinksTests({ getService }: FtrProviderContext) {
                 'enterpriseSearchContent',
                 'enterpriseSearchAnalytics',
                 'appSearch',
-                'workplaceSearch'
+                'workplaceSearch',
+                'guidedOnboarding'
```

Comment:
I'm not quite sure what is the correct approach to fix these tests

Reply:
@elastic/kibana-security Another question for the team: if you own these tests, could you please provide some guidance on what is being tested here and what is the best approach to fix those?

Reply:
Hi @yuliacech Sorry for the delay. The except method matches the feature ID to set the expected value to false. The feature ID is actually 'guidedOnboardingFeature', https://github.com/elastic/kibana/blob/d09c6cc5fa9629e79cacbb6a5aa9ce11ec5a9104/src/plugins/guided_onboarding/common/constants.ts#L14

This explains why it was still being set to true and those 3 tests were failing. I think you'll want to move the 'everything_space_all at everything_space' case. If I understand the feature/privileges correctly, this case should enable the navlink.